app/views/orders/index.html.erb

```diff
@@ -1,35 +1,30 @@
-<p id="notice"><%= notice %></p>
+<%= content_for :table_title do %>Orders<% end %>
 
-<h1>Orders</h1>
+<%= content_for :table_head do %>
+  <th>Product Name</th>
+  <th>Order Count</th>
+  <th>Customer</th>
+<% end %>
 
-<table>
-  <thead>
-    <tr>
-      <th>Product Name</th>
-      <th>Order Count</th>
-      <th>Customer</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @orders.each do |order| %>
+<%= content_for :table_body do %>
+  <% @orders.each do |order| %>
```

Comment:
Just head's up, this blows up sometimes if the `@orders` variable isn't set (which it isn't always). It probably is better to wrap an `if` statement around it.